src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -827,12 +829,16 @@ void hiopVectorRajaPar::startingAtCopyToStartingAt_w_pattern(index_type start_id
  * @param[out] dest - destination buffer where to copy vector data
  * 
  * @pre Size of `dest` must be >= n_local_
+ * @pre `dest` should be on the same memory space/backend as `this`
+ *
  * @post `this` is not modified
  */
-void hiopVectorRajaPar::copyTo(double* dest) const
+template<class MEM, class POL>
+void hiopVectorRaja<MEM,POL>::copyTo(double* dest) const
 {
-  auto& rm = umpire::ResourceManager::getInstance();
-  rm.copy(dest, this->data_dev_, n_local_*sizeof(double));
+  auto* This_const = const_cast<hiopVectorRaja*>(this);
+  assert(nullptr != This_const);
+  This_const->exec_space_.copy(dest, data_dev_, n_local_);
```

Comment:
```suggestion
  auto* this_const = const_cast<hiopVectorRaja*>(this);
  assert(nullptr != this_const);
  this_const->exec_space_.copy(dest, data_dev_, n_local_);
```
This would ideally be caught by a linter, but this is inconsistent with other variables.

Reply:
fixed